test/TensorFlow/debugging.swift

```diff
@@ -38,7 +47,14 @@ public func debugValuesInLoop(_ x: Tensor<Float>) {
 // CHECK:   br bb1
 
 
-// CHECK-LABEL: --- TFPartition Host Result: {{.*}}basicDebugValues{{.*}}
+// CHECK-LABEL: ---- ANALYSIS STATE FOR FUNCTION ${{.*}}noCopyForOpaqueHandle
+// CHECK: [Move]    %28 = graph_op "TensorSliceDataset,L,e"(%4 : $TensorHandle<Float>)
```

Comment:
Do we have to CHECK on `[Move]` here? So far `ANALYSIS STATE` is only used for human debugging. Would it be sufficient to CHECK on the output of `INPUT FUNCTION` here?

Reply:
Deleted that. Just a CHECK-NOT on the opaque handle in the list of tensor program results is enough. It just makes sure `debug_value` was ignored so that the opaque handle was never a tensor program result.